docs/csaf-vex.md

```diff
@@ -0,0 +1,497 @@
+# Red Hat CSAF-VEX Advisories
+
+## Red Hat Security Data Overview 
+The Red Hat Product Security was originally founded in 2001 and has always been committed to providing our customers 
+and partners with complete and accurate security data for all Red Hat software. In the past, Red Hat published security 
+advisory information using Common Vulnerability Reporting Framework (CVRF) and CVE information using the Open 
+Vulnerability and Assessment Language (OVAL) format. Over the last few years, the Common Security Advisory Framework 
+(CSAF) 2.0 standard was published and is now the successor to the CVRF version 1.2 as there are many enhancements to the
+information provided in each CSAF file. 
+
+On February 1st, 2023, Red Hat first announced the general availability of CSAF 2.0 documents. This version of our CSAF 
+files are published using the VEX profile (Vulnerability Exploitability eXchange) and the document type is now known as 
+csaf_vex. Since then Red Hat has continued to make improvements to our published security data, including the GA release
+of our VEX files on July 10th, 2024. While publishing our CSAF files was extremely helpful for correlating security data
+to individual advisories, we began publishing VEX files to make it easier for our partners and customers to correlate 
```

Comment:
If we really want to include this CSAF / VEX history, then it should be more clear that CSAF advisories are only for the product security updates (only one product new release that contains some vulnerability updates) when the VEX files for every single CVE id are focused on the particular CVE id and reflects how that vulnerability affects Red Hat portfolio (many products and their status for the single CVE).

Reply:
Removing this per Martin's comment: https://github.com/RedHatProductSecurity/security-data-guidelines/pull/29#discussion_r1772930342